docs/imaging_model/README.md

```diff
@@ -0,0 +1,47 @@
+# Imaging and Report Model
+
+## Motivation
+
+Participants often have files in addition to genomic files ranging from
+MRI images, histology images, and reports.
+Some of these file types relate directly to the biospecimen rather than the
+participant, eg: proteomics and histology.
+In order to capture and distribute these files, we need to model them
+appropriately.
+
+## Implementation
+
+Files will need to be distributed through gen3, so the models will need to
+inherit the Indexd model similar to GenomicFiles.
+
+![Imaging Model](ImagingModelTables.png)
+
+### ACLs
+
+These file types are typically not submitted to dbGaP and so can not be
+directly controlled through accession numbers and consents.
+Instead, these files will have ACLs consisting of the study kf_id and the
+most restricted set of consent codes for that participant's biospecimen.
+
+### Migration of current files
+
+Many MRI files have already been loaded as Genomic Files.
+These files will need to be moved to RadiologyFiles.
+Because this was intentionally done as a temporary solution, we will not make
+an attempt to preserve the kf_ids given to the files.
+
+
+## Difficulties
+
+These models and data files are currently only supported by CBTTC with
+only one additonal study in the future.
+Will this model be utilized at some point across more studies, or is it
+only being implemented to accomadate an outlier?
+
+The model changes introduce many more file types which will result in many
+more files being tracked in the dataservice and indexd.
+Indexd is currently a bottleneck, and will likely remain one.
+Will it be able to accommodate such an increase in quantity?
+
+
```

Comment:
Also discussed general report and event tables that could relate to any other table but will put off for later discussion.

Reply:
How is indexd a bottleneck, and why is it? 
Generally speaking, mapping one identifier to another should be trivial.

Reply:
We are not just mapping an identifier, though. We need to create documents within indexd and any interaction with files requires sub-requests to indexd.

Reply:
Sure, but indexd should be lightning fast because it's doing hardly anything.

Reply:
It's not terribly slow in performing it's logic, but it requires a web request per read and at least two http requests for everything else. This is a lot of overhead that slows it down when doing things in bulk.